src/MongoDB.Driver.Core/Core/Operations/GeoSearchOperation.cs

```diff
@@ -30,6 +30,7 @@ namespace MongoDB.Driver.Core.Operations
     /// Represents the geoSearch command.
     /// </summary>
     /// <typeparam name="TResult">The type of the result.</typeparam>
+    [Obsolete("The geoSearch was deprecated in server version 4.4.")]
```

Comment:
Replace this Obsolete message with this one that reads better:

    [Obsolete("The geoSearch command was deprecated in server version 4.4.")]

See next comment for a rewording of all other new Obsolete message.

Reply:
done